addons/addons.json

```diff
@@ -1,4 +1,5 @@
 [
+  "costume-editor-shortcuts",
```

Comment:
New addons are usually inserted above `"// NEW ADDONS ABOVE THIS ↑↑",` near the end of this file.